Dart/thirdPartySrc/analysisServer/com/google/dart/server/internal/remote/RemoteAnalysisServerImpl.java

```diff
@@ -570,6 +578,12 @@ public void server_getVersion(GetVersionConsumer consumer) {
     sendRequestToServer(id, RequestUtilities.generateServerGetVersion(id), consumer);
   }
 
+  // This is overridden due to the generated methods from the Dart SDK, but the implementation (if implemented) would be in #processRequests.
+  @Override
+  public void server_openUrlRequest(String url) {
+    // do nothing.
+  }
```

Comment:
I declared this `RemoteAnalysisServerImpl` class abstract and removed empty `server_openUrlRequest` and `server_showMessageRequest` implementations. This way I had to implement them right in the Dart plugin, where I create and instance of `RemoteAnalysisServerImpl`. So it allowed me not to use any IntelliJ-specific classes here, but to delegate the IntelliJ-specific implementation right to the Dart plugin.